bin/upload-docs

```diff
@@ -2,22 +2,24 @@
 set -ef -o pipefail
 
 bucket="chain-staging.chain.com"
-if [ "$1" = "prod" ];
+if [ "$2" = "prod" ];
 then
   bucket="chain.com"
 fi
 
 bucket_url=s3://$bucket
 
+git checkout $1-stable
```

Comment:
This also has an unexpected side effect of leaving your git repo checked out to a different repository than you started with. I think it's worth it to us to copy the `build-cored-release` system of cloning the git repo to a tmp directory, and doing our checkout and builds there:

https://github.com/chain/chain/blob/main/bin/build-cored-release#L28-L36